pkg/inspector/image-inspector.go

```diff
@@ -133,15 +142,48 @@ func (i *defaultImageInspector) Inspect() error {
 		} else {
 			i.meta.OpenSCAP.Status = iiapi.StatusSuccess
 		}
+		scanResults.Results = append(scanResults.Results, openscap.ParseResults(scanReport)...)
```

Comment:
@enoodle we also want to serve this via HTTP, see https://github.com/openshift/image-inspector/pull/55